MainRepository/Program.cs

```diff
@@ -1,7 +1,30 @@
-﻿internal class Program
-{
-    private static void Main(string[] args)
-    {
-        Console.WriteLine("Hello, World!");
-    }
+﻿// Program to find guess number from 1 to 127.
+namespace Test2 {
+   class GuessNumber {
+      public static void GuessGame () {
+         int n = 2;

```

Comment:
Declare variable right before where it is used. In this case, it should be at line 11.